checkov.yaml

```
@@ -1,6 +1,6 @@
 package:
   name: checkov
-  version: 2.3.354
+  version: 3.0.24
   epoch: 1
```

```suggestion
  epoch: 0
```
In version upgrade epoch should be bump down to 0